modules/cnv-example-kubevirtstorageclassdefaults.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_virtual_machines/cnv_virtual_disks/cnv-storage-defaults-for-datavolumes.adoc
+
+[id="cnv-example-kubevirtstorageclassdefaults_{context}"]
+= Example of multiple storage class defualts:
+
+The following YAML file is an example of a `kubevirt-storage-class-defaults` ConfigMap that has storage settings configured for two storage classes, `migration` and `block`. 
+If you edit the default storage settings, or add settings for a storage class, ensure these are supported by the underlying storage.
+
+[source,yaml]
+----
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: kubevirt-storage-class-defaults
+  namespace: openshift-cnv
+...
+data:
+  accessMode: ReadWriteOnce
+  volumeMode: Filesystem
+  <migration>-sc.accessMode: ReadWriteMany 
+  <migration>-sc.volumeMode: Filesystem
+  <block>-sc.accessMode: ReadWriteMany
+  <block>-sc.volumeMode: Block
```

Comment:
Let's use realistic storage class names for the example (no <>).  I recommend the following:
- nfs-sc
- block-sc